Qualys_Add_New_Commands (#31917)
demisto/content: Packs/qualys/Integrations/Qualysv2/README.md

#### Human Readable Output

>Asset tag deleted.

### qualys-update-vmware-record

***
Update Vmware records for authenticated scans of hosts running on Vmware.

#### Base Command

`qualys-update-vmware-record`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |

> | ids | Record IDs to update. Specify record IDs and/or ID ranges Multiple entries are comma separated. | Required | 
> | add_ips | Add IPs and/or ranges to the IPs list for this record. Multiple IPs/ranges are comma separated. | Required | 

Reviewer: ```suggestion
| ids | A comma-separated list of record IDs to update. Specify record IDs and/or ID ranges. | Required | 
| add_ips | A comma-separated list of IPs and/or ranges to add to the IPs list for this record. | Required | 
```

#### Context Output

There is no context output for this command.

#### Command Example
```!qualys-update-vmware-record ids=123 add_ips=5.2.8.9```

#### Human Readable Output

>Successfully Updated

### qualys-update-vcenter-record

***
Update vCenter records for authenticated scans of hosts running on vCenter.

#### Base Command

`qualys-update-vcenter-record`

#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |

> | ids | Record IDs to update. Specify record IDs and/or ID ranges Multiple entries are comma separated. | Required | 
> | add_ips | Add IPs and/or ranges to the IPs list for this record. Multiple IPs/ranges are comma separated. | Required | 

Reviewer: ```suggestion
| ids | A comma-separated list of record IDs to update. Specify record IDs and/or ID ranges. | Required | 
| add_ips | A comma-separated list of IPs and/or ranges to add to the IPs list for this record. | Required | 
```
